src/ManageDatesView.elm

```diff
@@ -33,44 +33,52 @@ view model =
             model.pickedDates
                 |> List.indexedMap
                     (\i d ->
-                        div []
-                            [ input
-                                [ class "form-control pikaday-input"
-                                , id ("pikaday-instance-" ++ (toString i))
-                                , value (displayDateWithoutTime d)
-                                , disableInput d
+                        div [ class "form__question-section form__question-section--table" ]
```

Comment:
Please can we _also_ make this one:
`form__question-section form__question-section--table form__question-section--table-auto`
so that it matches the others